src/c4/runtime_check.cc

```diff
@@ -0,0 +1,36 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   ds++/runtime_check.cc
+ * \author Kent Grimmett Budge
```

Comment:
When you move these files to `diagnostics` please globally replace "ds++" with "diagnostics". Ditto for "dsxx" (namespace, etc.).